cmd/serverless/main.go

```diff
@@ -69,15 +71,18 @@ const (
 
 func main() {
 	// run the agent
-	err := fxutil.OneShot(runAgent)
+	err := fxutil.OneShot(
+		runAgent,
+		taggernoop.Module(),
```

Comment:
Serverless uses it own logs agent. I wasn't sure if serverless need the real tagger or not. To avoid adding any extra dependency I created a noop implementation.

cc @DataDog/serverless